from src import *
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainMenuWIndow(object):
    def setupUi(self, MainMenuWIndow):
        MainMenuWIndow.setObjectName("MainMenuWIndow")
        MainMenuWIndow.resize(500, 300)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(MainMenuWIndow.sizePolicy().hasHeightForWidth())
        MainMenuWIndow.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setKerning(True)
        MainMenuWIndow.setFont(font)
        MainMenuWIndow.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        MainMenuWIndow.setMouseTracking(False)
        MainMenuWIndow.setAcceptDrops(False)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("program\src\icon\icon.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        MainMenuWIndow.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(MainMenuWIndow)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.centralwidget.sizePolicy().hasHeightForWidth())
        self.centralwidget.setSizePolicy(sizePolicy)
        self.centralwidget.setObjectName("centralwidget")
        self.menu1_btn = QtWidgets.QPushButton(self.centralwidget)
        self.menu1_btn.setGeometry(QtCore.QRect(110, 70, 281, 51))
        self.menu1_btn.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.menu1_btn.setObjectName("menu1_btn")
        self.title_label = QtWidgets.QLabel(self.centralwidget)
        self.title_label.setGeometry(QtCore.QRect(10, 0, 481, 51))
        font = QtGui.QFont()
        font.setPointSize(16)
        font.setBold(True)
        font.setWeight(75)
        font.setKerning(True)
        self.title_label.setFont(font)
        self.title_label.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        self.title_label.setMouseTracking(False)
        self.title_label.setTextFormat(QtCore.Qt.AutoText)
        self.title_label.setAlignment(QtCore.Qt.AlignCenter)
        self.title_label.setObjectName("title_label")
        self.menu2_btn = QtWidgets.QPushButton(self.centralwidget)
        self.menu2_btn.setGeometry(QtCore.QRect(110, 130, 281, 51))
        self.menu2_btn.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.menu2_btn.setObjectName("menu2_btn")
        self.menu3_btn = QtWidgets.QPushButton(self.centralwidget)
        self.menu3_btn.setGeometry(QtCore.QRect(110, 190, 281, 51))
        self.menu3_btn.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.menu3_btn.setObjectName("menu3_btn")
        MainMenuWIndow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainMenuWIndow)
        self.menubar.setEnabled(True)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 500, 21))
        self.menubar.setObjectName("menubar")
        self.menuInfo = QtWidgets.QMenu(self.menubar)
        self.menuInfo.setObjectName("menuInfo")
        MainMenuWIndow.setMenuBar(self.menubar)
        self.action_github = QtWidgets.QAction(MainMenuWIndow)
        self.action_github.setCheckable(False)
        self.action_github.setEnabled(True)
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("program\src\icon\github.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.action_github.setIcon(icon1)
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        self.action_github.setFont(font)
        self.action_github.setObjectName("action_github")
        self.menuInfo.addAction(self.action_github)
        self.menubar.addAction(self.menuInfo.menuAction())

        self.retranslateUi(MainMenuWIndow)
        QtCore.QMetaObject.connectSlotsByName(MainMenuWIndow)

    def retranslateUi(self, MainMenuWIndow):
        _translate = QtCore.QCoreApplication.translate
        MainMenuWIndow.setWindowTitle(_translate("MainMenuWIndow", "program_title"))
        self.menu1_btn.setText(_translate("MainMenuWIndow", "File Encryption"))
        self.title_label.setText(_translate("MainMenuWIndow", "program_title"))
        self.menu2_btn.setText(_translate("MainMenuWIndow", "File Decryption"))
        self.menu3_btn.setText(_translate("MainMenuWIndow", "Program Exit"))
        self.menuInfo.setTitle(_translate("MainMenuWIndow", "Info"))
        self.action_github.setText(_translate("MainMenuWIndow", "github"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    MainMenuWIndow = QtWidgets.QMainWindow()
    ui = Ui_MainMenuWIndow()
    ui.setupUi(MainMenuWIndow)
    MainMenuWIndow.show()
    sys.exit(app.exec_())
